src/index.ts

```diff
@@ -0,0 +1,62 @@
+// FIXME: lint fails
+/* eslint-disable no-underscore-dangle */
+import http = require('http');
+import https = require('https');
```

Comment:
`import = require` should be changed to `import * as x`

Reply:
`import * as QuickLRU from 'quick-lru';` doesn't work as it also exports a namespace